ONBOARDING.md

```diff
@@ -0,0 +1,139 @@
+## Getting Started
+
+### Core SDK
+
+First access the mParticle Core SDK repository via the following
+link: [mParticle GitHub Repo](https://github.com/mParticle/mparticle-android-sdk)
+You can easily "fork" the repository by clicking the **Fork** tab in the upper right corner, once
+that is done you should be able to see it in your own Github account under the **Repositories** tab.
+Open your forked version and in the green **Code** tab copy the https link. Go ahead and open
+Android Studio and select Project from Version Control or Get from VCS, paste the Repository link
+you just copied and select **Clone**.
+
+### Testing the Core SDK
+
+Once the project is loaded, run the default tests to validate that your environment is set up
+correctly and to check that the code is working correctly before making changes of your own.
+
+You can do so locally using the the following gradle wrapper commands in your terminal:
+
+For Lint Checks:
+`./gradlew lint`
+
+For Unit Tests:
+`./gradlew test`
+
+For Instrumented Tests:
+`./gradlew :android-core:cAT :android-kit-base:cAT --stacktrace`
+
+In some machines running windows you may need to type the commands without **./**
+eg. `gradlew lint`
+
+#### Issues with JAVA_HOME
+
+When running the tests for the first time, It is possible that you run into the following issue:
+
+**ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.**
+
+**Please set the JAVA_HOME variable in your environment to match the location of your Java
+installation.**
+
+Any version between Java 8 and 17 is required to execute gradle, **Java 18 and later versions are
```

Comment:
highest is JDK 11 i think?  cc: @willpassidomo what's the highest version you're seeing?